sound/soc/sof/ipc4-topology.c

```diff
@@ -815,6 +817,74 @@ static void sof_ipc4_widget_free_comp_mixer(struct snd_sof_widget *swidget)
 	swidget->private = NULL;
 }
 
+/*
+ * Add the process modules support. The process modules can be EQFIR, EQIIR,
+ * KEYWORD_DETECT, KPB (KeyPhrase Buffer Manager), CHAN_SELECTOR, SMART_AMP
```

Comment:
do you mean snd_soc_dapm_effect by process comp? if so, you should mention it instead of the examples, you will never exhaust snd_soc_dapm_effect by examples.

Reply:
OK, will update it.

Reply:
If we use effect type, it seems not accurate. Because MUX is defined in effect but it is not supported in effect, it is treated as a snd_soc_dapm_mux widget.

Reply:
it is ok to use snd_soc_dapm_effect and snd_soc_dapm_mux if both are supported.